--- original-ks.cfg	2020-04-21 07:48:31.549476426 +0900
+++ c7rpi-ks.cfg	2020-05-08 00:34:35.608857128 +0900
@@ -3,40 +3,39 @@
 install
 # Keyboard layouts
 # old format: keyboard us
-# new format:
-keyboard --vckeymap=us --xlayouts='us'
+# new format: keyboard jp
+keyboard --vckeymap=jp --xlayouts='jp'
 # Root password
 rootpw --plaintext centos
 # System language
-lang en_US.UTF-8
+lang ja_JP.UTF-8
 # SELinux configuration
 selinux --enforcing
 
 # System services
-services --enabled="sshd,NetworkManager,chronyd"
+services --enabled="sshd,NetworkManager,chronyd,tuned"
 
 # Firewall configuration
-firewall --enabled --port=22:tcp
+firewall --enabled --service=ssh --service=cockpit
 # Network information
 network  --bootproto=dhcp --device=link --activate
 # Shutdown after installation
 shutdown
-repo --name="instCentOS" --baseurl=http://qa.centos.org/altarch/7/os/armhfp/ --cost=100
-repo --name="instUpdates" --baseurl=http://qa.centos.org/altarch/7/updates/armhfp/ --cost=100
-repo --name="instExtras" --baseurl=http://qa.centos.org/altarch/7/extras/armhfp/ --cost=100
-repo --name="instKern" --baseurl=http://qa.centos.org/altarch/7/kernel/armhfp/kernel-rpi2/ --cost=100
+repo --name="instCentOS"  --baseurl=http://mirror.centos.org/altarch/7/os/armhfp/                 --cost=100
+repo --name="instUpdates" --baseurl=http://mirror.centos.org/altarch/7/updates/armhfp/            --cost=10
+repo --name="instExtras"  --baseurl=http://mirror.centos.org/altarch/7/extras/armhfp/             --cost=100
+repo --name="instKern"    --baseurl=http://mirror.centos.org/altarch/7/kernel/armhfp/kernel-rpi2/ --cost=100
 # System timezone
-timezone UTC --isUtc --nontp
+timezone Asia/Tokyo --isUtc --nontp
 # Use network installation
-url --url="http://qa.centos.org/altarch/7/os/armhfp/"
+url --url="http://mirror.centos.org/altarch/7/os/armhfp/"
 # System bootloader configuration
 bootloader --location=mbr
 # Partition clearing information
 clearpart --all --initlabel
 # Disk partitioning information
-part /boot --asprimary --fstype="vfat" --size=300 --label=boot
-part swap --asprimary --fstype="swap" --size=512 --label=swap
-part / --asprimary --fstype="ext4" --size=1800 --label=rootfs
+part /boot --asprimary --fstype="ext4" --size=100 --label=boot
+part / --asprimary --fstype="ext4" --size=1947 --label=rootfs
 
 %post
 # Mandatory README file
@@ -48,11 +47,6 @@
 
 EOF
 
-# Enabling chronyd on boot
-systemctl enable chronyd
-
-
-
 # Remove ifcfg-link on pre generated images
 rm -f /etc/sysconfig/network-scripts/ifcfg-link
 
@@ -65,7 +59,7 @@
 %post
 # Specific cmdline.txt files needed for raspberrypi2/3/4
 cat > /boot/cmdline.txt << EOF
-console=ttyAMA0,115200 console=tty1 root=/dev/mmcblk0p3 rootfstype=ext4 elevator=deadline rootwait
+console=ttyAMA0,115200 console=tty1 root=/dev/mmcblk0p2 rootfstype=ext4 elevator=deadline rootwait
 EOF
 
 # Setting correct yum variable to use raspberrypi kernel repo
@@ -342,3 +336,7 @@
 -python*-caribou*
 
 %end
+
+%include ../ksmodules/c7rpi-miscs.ksmod
+%include ../ksmodules/c7rpi-fswap.ksmod
+%include ../ksmodules/c7rpi-cockpit.ksmod
